src/MICore/MICore.csproj

```diff
@@ -199,7 +200,8 @@
   <ItemGroup>
     <GlassDirCopy Include="$(OutDir)$(AssemblyName)$(TargetExt)" />
     <GlassDirCopy Include="$(OutDir)$(AssemblyName).XmlSerializers.dll" Condition="'$(Configuration)'!='Debug-PortablePDB'" />
-    <GlassDirCopy Include="$(OutDir)Newtonsoft.Json.dll" />
+	<GlassDirCopy Include="$(NuGetPackagesDirectory)\Newtonsoft.Json\6.0.8\lib\portable-net45+wp80+win8+wpa81+aspnetcore50\Newtonsoft.Json.dll" Condition="'$(IsCoreClr)' == 'true'" />
```

Comment:
There is a one character difference between the two lines -- `\` vs `.` separating the version from the package name.